packages/@react-aria/grid/src/useGrid.ts

```diff
@@ -95,7 +95,8 @@ export function useGrid<T>(props: GridProps, state: GridState<T, GridCollection<
     selectionManager: state.selectionManager,
     keyboardDelegate: delegate,
     isVirtualized,
-    scrollRef
+    scrollRef,
+    disableNavigation: state.disableNavigation
```

Comment:
Does this need to be added to the useSelectableCollection hook, or could we just ignore `collectionProps` if disabled?

Reply:
I think it does stay because of the handling of Tab key https://github.com/adobe/react-spectrum/pull/3258/files#diff-e356ae602508922357fd9cd1aa7896f74e24f87bcb2ee86a7bb4a486ea87a2f2R235

Reply:
Hmm ok, i suppose that makes sense then, including link for posterity
https://www.w3.org/WAI/ARIA/apg/patterns/grid/